conf/vsc_kul_uhasselt.config

```diff
@@ -56,112 +44,137 @@ env {
 
 // AWS maximum retries for errors (This way the pipeline doesn't fail if the download fails one time)
 aws {
-        maxErrorRetry = 3
+    maxErrorRetry = 3
 }
 
 // Define profiles for each cluster
 profiles {
     genius {
-        params {
-            config_profile_description = 'HPC_GENIUS profile for use on the genius cluster of the VSC HPC.'
-            config_profile_contact = 'GitHub: @Joon-Klaps - Email: joon.klaps@kuleuven.be'
-            config_profile_url = 'https://docs.vscentrum.be/en/latest/index.html'
-            max_memory = 703.GB  // 768 - 65 so 65GB for overhead, max is 720000MB
-            max_time = 168.h
-            max_cpus = 36
-        }
+        params.config_profile_description = 'genius profile for use on the genius cluster of the VSC HPC.'
 
         process {
-            resourceLimits = [
-                memory: 703.GB,
-                cpus: 136,
-                time: 168.h
-            ]
-            executor = 'slurm'
+            // 768 - 65 so 65GB for overhead, max is 720000MB
+            resourceLimits = [ memory: 703.GB, cpus: 36, time: 168.h ]
+            beforeScript = 'module load cluster/genius'
+            clusterOptions = { "--clusters=genius --account=$tier1_project" }
+
             queue = {
-                switch (task.memory) {
-                case { it >=  175.GB }: // max is 180000
-                    switch (task.time) {
-                    case { it >= 72.h }:
-                        return 'dedicated_big_bigmem,dedicated_big_batch,bigmem_long'
-                    default:
-                        return 'bigmem'
-                    }
-                default:
-                    switch (task.time) {
-                    case { it >= 72.h }:
-                        return 'batch_long'
-                    default:
-                        return 'batch'
-                    }
+                task.memory >= 175.GB ?
+                    (task.time >= 72.h ? 'dedicated_big_bigmem,dedicated_big_batch,bigmem_long' : 'bigmem') :
+                    (task.time >= 72.h ? 'batch_long' : 'batch')
+            }
+
+            withLabel: '.*gpu.*'{
```

Comment:
Maybe restrict this to `gpu` only, I can see some edge cases come up if you do it like this

Reply:
mmmh... but what if non nf-core pipelines use other labels like 'process_gpu' or 'gpu_job'? @tatianawoller any  thoughts?

Reply:
It's fine for me but I can see this breaking other things where the word `gpu` is also in the label

Reply:
I see what you mean but which label would have the string 'gpu' in them without referring to a GPU? Or am I overlooking something?

Reply:
It probably won't happen but it can also fail when the letters `gpu` are inside another word or name, but I'll approve this, you can still fix this later if you were to encounter this issue

Reply:
Yeah, ok. If issues occur we'll patch